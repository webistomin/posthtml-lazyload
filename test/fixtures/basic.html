<img src="http://posthtml.github.io/posthtml/logo.svg" alt="PostHTML" width="220" height="200" class="lazyload">
<img src="http://posthtml.github.io/posthtml/logo.svg" alt="PostHTML" width="220" height="200" class="no-lazyload">
