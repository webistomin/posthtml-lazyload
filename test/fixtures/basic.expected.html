<img src="http://posthtml.github.io/posthtml/logo.svg" alt="PostHTML" width="220" height="200" class="lazyload" loading="eager">
<img src="http://posthtml.github.io/posthtml/logo.svg" alt="PostHTML" width="220" height="200" class="no-lazyload">
